server/app/repository/ProgramRepository.java

```diff
@@ -46,6 +46,7 @@
  */
 public final class ProgramRepository {
   private static final Logger logger = LoggerFactory.getLogger(ProgramRepository.class);
+  private final QueryUtils queryUtils = new QueryUtils("ProgramRepository");
```

Comment:
nit (non-blocking, optional): `QueryUtils` feels like it'd be a static class. Is there a name for this that might not imply that? Something like `QueryProfileLocationBuilder`?

Or maybe `QueryUtils` is static and here we can call a `getProfileLocationBuilder()` method on it?

Reply:
It can be a static variable here (updated to add), but also renamed to QueryProfileLocationBuilder, since that better represents what its doing. Let me know what you think!